src/.htaccess

```diff
@@ -16,6 +16,10 @@ Options All -Indexes
 # Define time zone
 SetEnv TZ Europe/Paris
 
+# File upload sizes
+php_value upload_max_filesize 10M
+php_value post_max_size 0
```

Comment:
Isn't post_max_size a bit risky, as someone could send a 1Gb request out of many 10M photos ?

Reply:
Yup, you have a point. I set it to 50M. What do you think ?

Reply:
Good for me!